test/cbmc/proofs/Makefile-template-defines

```diff
@@ -2,7 +2,7 @@
 # Absolute path to the root of the source tree.
 #
 SRCDIR ?= $(abspath $(PROOF_ROOT)/../../../)
-
+>
```

Comment:
What is this character? Was this added by accident?

Reply:
Removed it in 207cf11